test/connections/databases.ci.yml

```diff
@@ -20,3 +20,13 @@ sqlite:
   adapter: sqlite3
   database: <%= File.join(project_root, 'db', 'composite_primary_keys_unittest.sqlite') %>
 
+trilogy:
+  adapter: mysql2
```

Comment:
The adapter isn't trilogy?

Reply:
You're right. Not sure why I had this. I'll use trilogy 😅